### PR TITLE
QF-1506 Add 7th July banner on start and home pages

### PR DESCRIPTION
**Pull request by SawNawDfE**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,14 +1,17 @@
-﻿@model HomeIndexViewModel
+﻿@using SFA.DAS.AssessorService.Web.Helpers;
+@model HomeIndexViewModel
 @{
     ViewData["Title"] = "Apprenticeship assessment service";
     ViewBag.ZenDeskLabel = "ass-start-page";
     Layout = "_Layout";
+    Model.Banner = new Banner("~/Views/Banners/_NoAssessBannerPartial.cshtml");
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/Home/Index.cshtml:7]: This should be set where the model is created in the controller

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AssessorService.Web.UnitTests/Helpers/BannerTests.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using NUnit.Framework;
+using SFA.DAS.AssessorService.Web.Helpers;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.AssessorService.Web.UnitTests.Helpers;
+
+public class BannerTests
+{
+    [Test, MoqAutoData]
+    public void ViewPath_ReturnsValueSetInConstructor(string path)
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Web.UnitTests/Helpers/BannerTests.cs]: Banner logic reworked and tests added for the two view models

File: src/SFA.DAS.AssessorService.Web/Helpers/Banner.cs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+﻿namespace SFA.DAS.AssessorService.Web.Helpers;
+
+public readonly record struct Banner(string ViewPath);
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Web/Helpers/Banner.cs]: This was part of an attempt to account for the dates which didn't make it in the final code. I've refactored this into a constant string that is referenced by the two view models that are used to display the banner

File: src/SFA.DAS.AssessorService.Web/Views/Dashboard/Index.cshtml [hunk 4]
```diff
@@ -1,10 +1,12 @@
 ﻿@using SFA.DAS.AssessorService.Web.Infrastructure
+@using SFA.DAS.AssessorService.Web.Helpers;
 @model SFA.DAS.AssessorService.Web.ViewModels.Dashboard.DashboardViewModel
 @inject ISessionService SessionService
 @{
     ViewData["Title"] = "Home";
     ViewBag.ZenDeskLabel = "ass-dashboard";    
     Layout = "_DashboardLayout";    
+    Model.Banner = new Banner("~/Views/Banners/_NoAssessBannerPartial.cshtml");
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Web/Views/Dashboard/Index.cshtml]: Reworked as mentioned above

File: src/SFA.DAS.AssessorService.Web/Views/Home/Index.cshtml (hunk 1 above)
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Web/Views/Home/Index.cshtml]: Reworked as mentioned above

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**